### PR TITLE
178 students can't edit plan

### PR DESCRIPTION
**Pull request by yurakozlovskiy**

### CODE REVIEW

File: src/app/plan/plan-details/plan-details.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <app-tasks [planId]="plan.Id"></app-tasks>
 <mat-card-actions>
-    <button mat-button (click)="openEditDialog()">
+    <button *ngIf="isMentor" mat-button (click)="openEditDialog()">
```
Comment (YuLad) [inline src/app/plan/plan-details/plan-details.component.html]: Admin should also be capable of editing plans

File: src/app/plan/plan-details/plan-details.component.ts [hunk 2]
```diff
@@ -12,9 +13,13 @@ export class PlanDetailsComponent implements OnInit {
 
   @Input()
   plan: Plan;
-  constructor(public dialog: MatDialog) { }
+  isMentor=false;
```
Comment (YuLad) [inline src/app/plan/plan-details/plan-details.component.ts]: Use linter and fix formatting

File: src/app/task/task-detail/task-detail.component.html [hunk 3]
```diff
@@ -26,13 +26,13 @@ <h3>{{task.Description}}</h3>
     </mat-expansion-panel>    
     <div>
       <mat-card-actions>
-        <button mat-button (click)="openEditDialog()">
+        <button *ngIf="isMentor"  mat-button (click)="openEditDialog()">
```
Comment (YuLad) [inline src/app/task/task-detail/task-detail.component.html]: Admins should also have rights for this

File: src/app/task/task-detail/task-detail.component.ts [hunk 4]
```diff
@@ -14,10 +15,13 @@ export class TaskDetailComponent implements OnInit {
 
   @Input()
   task: Task;
-  constructor(public dialog: MatDialog) {
-  }
+  isMentor=false;
+  constructor(public dialog: MatDialog,private authService:AuthService) { }
```
Comment (YuLad) [inline src/app/task/task-detail/task-detail.component.ts]: formatting

File: src/app/task/task-detail/task-detail.component.ts [hunk 5]
```diff
@@ -14,10 +15,13 @@ export class TaskDetailComponent implements OnInit {
 
   @Input()
   task: Task;
-  constructor(public dialog: MatDialog) {
-  }
+  isStudent = false;
+  constructor(public dialog: MatDialog, private authService: AuthService) { }
 
   ngOnInit() {
+    if (this.authService.isMentor()) {
```
Comment (YuLad) [inline src/app/task/task-detail/task-detail.component.ts]: this is so wrong; fix

Comment (yurakozlovskiy) [inline src/app/task/task-detail/task-detail.component.ts]: Sorry,didn't look carefully.I've fixed them

**Pull request closed without merging**